app/src/main/java/asd/org/ahorcado/activities/UtilActivity.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Muber 2016. Copyright © All rights reserved.
+ */
+package asd.org.ahorcado.activities;
+
+import android.view.View;
+import android.widget.RelativeLayout;
+
+import asd.org.ahorcado.R;
+
+public class UtilActivity {
```

Comment:
I think we could rename this class something like "ImageHelper" and put in a helper. Also we need to try to avoid the static call